src/Components/Footer/FooterDownloadAppBanner.tsx

```diff
@@ -16,58 +16,57 @@ const MOBILE_COVER_IMAGE = resized(
-    <Box borderBottom="1px solid" borderColor="black10">
-      <GridColumns>
-        <Column
-          span={4}
-          display="flex"
-          flexDirection="column"
-          alignItems="center"
-          justifyContent="center"
-          order={[2, 1]}
-          px={2}
-          py={[6, 2]}
-        >
-          <Text variant="xl" textAlign="center" mb={1}>
-            Get the Artsy app
-          </Text>
+    <GridColumns borderBottom="1px solid" borderColor="black10">
+      <Column
+        span={4}
+        display="flex"
+        flexDirection="column"
+        alignItems="center"
+        justifyContent="center"
+        order={[2, 1]}
+        px={2}
+        py={[6, 2]}
+      >
+        <Text variant="xl" textAlign="center" mb={1}>
+          Get the Artsy app
+        </Text>
 
-          <Spacer mt={2} />
+        <Spacer mt={2} />
 
-          <DownloadAppBadges contextModule={ContextModule.footer} />
-        </Column>
+        <DownloadAppBadges contextModule={ContextModule.footer} />
+      </Column>
 
-        <Column span={8} position="relative" order={[1, 2]}>
-          <Media at="xs">
-            <Image
-              src={MOBILE_COVER_IMAGE.src}
-              srcSet={MOBILE_COVER_IMAGE.srcSet}
-              height={320}
-              width="100%"
-              lazyLoad
-              alt=""
-              style={{ objectFit: "cover", objectPosition: "center top" }}
-            />
-          </Media>
+      <Column span={8} position="relative" order={[1, 2]}>
+        <Media at="xs">
+          <Image
+            src={MOBILE_COVER_IMAGE.src}
+            srcSet={MOBILE_COVER_IMAGE.srcSet}
+            height={320}
+            width="100%"
+            lazyLoad
+            alt=""
+            style={{ objectFit: "cover", objectPosition: "center top" }}
+          />
+        </Media>
 
-          <Media greaterThan="xs">
-            <Image
-              src={DESKTOP_COVER_IMAGE.src}
-              srcSet={DESKTOP_COVER_IMAGE.srcSet}
-              height={320}
-              width="100%"
-              lazyLoad
-              alt=""
-              style={{ objectFit: "cover", objectPosition: "center top" }}
-            />
-          </Media>
-        </Column>
-      </GridColumns>
-
-      <Text variant="xs" fontStyle="italic" textAlign="right" my={1}>
-        Nicolas Party, Trees, 2019. Damien Cifelli, May You Live in Interesting
-        Times, 2022. Anna Park, It's Good For You, 2020
-      </Text>
-    </Box>
+        <Media greaterThan="xs">
+          <Image
+            src={DESKTOP_COVER_IMAGE.src}
+            srcSet={DESKTOP_COVER_IMAGE.srcSet}
+            height={320}
+            width="100%"
+            lazyLoad
+            alt=""
+            style={{ objectFit: "cover", objectPosition: "center top" }}
+          />
+        </Media>
+      </Column>
+      <Column span={12} order={[1, 2]}>
+        <Text variant="xs" fontStyle="italic" textAlign="right" mt={-10} pb={1}>
```

Comment:
This negative margin was the only way i could figure out how to remove the row gap.

Reply:
Went this way because the `order` prop was messing up the layout on mobile

Reply:
Just use a responsive rowGap?

Reply:
The `pb={1}` is also weird and I would avoid it. If you want external space, find the parent.